src/popup/Settings.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useCallback, useEffect, useState } from 'react'
-import _ from 'lodash'
+import _ from 'underscore'
```

Comment:
This is beyond my understanding. Isn't lodash supposed to replace underscore? Has underscore surpassed lodash now? 🤔

Reply:
not sure, I haven't found another way yet.😅
related: https://github.com/lodash/lodash/issues/4985

Reply:
It looks like this is an overkill. Only equal (deep equal) is being used. We can use some dedicated library like https://www.npmjs.com/package/fast-equals instea.

Reply:
@pengx17 That makes sense. I'll merge it first on my end and then use this library!